packages/fxa-auth-server/scripts/paypal-processor.ts

```diff
@@ -101,6 +102,7 @@ export async function init() {
       throw new Error(`Cannot acquire lock to run: ${err.message}`);
     }
   } else {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     for await (const _ of processor.processInvoices()) {
       // no need to do anything between invoices since we are not extending a lock
     }
```

Comment:
Not part of this PR, but I'm curious why we have this `for await..of` loop at all? Perhaps @chenba might be able to provide some insight as the author?

Reply:
It's so that processor.processInvoices() yields until it's done.

processor.processInvoices() was written in that way because we want to check the lock between each yield; see line 95.  Here there's no lock so we just have an empty block for the for..of.